docs/concepts/distributed_trust.md

```diff
@@ -0,0 +1,67 @@
+---
+id: distributed_trust
+title: Distributed Trust
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+:::caution
+This feature is currently a preview on Peregrine and is therefore not available on Spiritnet.
+:::
+
+Attesters can be individuals that attest to the validity of claims made by claimers.
+But in many cases multiple attesters group together to build up trust in a brand.
+Verifier don't need to trust individual attesters but trust the brand, which in return ensures that all attesters working for this brand are credible.
+Such a brand can be organized in many different ways.
+The KILT protocol provides mechanisms to form such brands on the blockchain.
+
+There are two ways for attesters to group up and build an organization.
+The first one are Delegation Hierarchies, which provide a very basic hierarchical organisation.
+Delegation Hierarchies are available on our test network Peregrine.
+The second option are Virtual Credential Organizations which are not yet implemented and currently.
+Virtual Credential Organizations will be more flexible than Delegation Hierarchies and accommodate a more decentralized range of use cases.
+
+## Delegation Hierarchy
+
+Delegation Hierarchies organize their members in a traditional hierarchical structure.
+Every one can create a new hierarchy and imminently become the only member of the organization.
+Not only is the creator the only member but also the root of the hierarchy.
+They have full control over the Delegation Hierarchy.
+
+When the root adds new attesters to the hierarchy, these new members become direct descendants of the root.
+The root is the parent of the new members while the members are the children of the root.
+When the children add members themselves, the children become parents too.
+This creates a tree like structure similar to the one shown in the image below.
+
+<center>
+
+<ThemedImage
+  alt="Example hierarchy"
+  sources={{
+    light: '/img/concepts/distributed_trust/delegation-hierarchies.svg',
+    dark: '/img/concepts/distributed_trust/delegation-hierarchies-dark.svg',
+  }}
+/>
+
+</center>
+
+The above depicted Delegation Hierarchy contains five attesters.
+Attester 1 is the root of the Delegation Hierarchy.
+As the root Attester 1 added two more attesters.
+Attester 2 was given the right to delegate and to issue attestations for the collective.
+Attester 3 can only add more attesters to the Delegation Hierarchy, but he cannot create, revoke or remove Attestations.
+Attesters 4 and 5 where added by Attester 3 and can only create, revoke and remove attestations, but they cannot add more attesters to the Delegation Hierarchy.
```

Comment:
I would add an example also for this case, which will probably be more common that the one above. For instance you might want someone to attest on your behalf, if you are on holiday, without the possibility of extending the delegation hierarchy before you return.

Reply:
Yes or in a company, you want that the employees who do the attestation work are not able to add people to the company. Only HR or the manager should be able to "hire" new employees.

Reply:
[d5af837](https://github.com/KILTprotocol/docs/pull/88/commits/d5af8378b4d9d20a68b2299fdef41fc3230bde4f)